omero/sysadmins/unix/install-web/apache-wsgi-omero.conf

```diff
@@ -56,7 +56,7 @@
 
   DocumentRoot /home/omero/OMERO.server/lib/python/omeroweb
 
-  WSGIDaemonProcess omeroweb processes=5 threads=1 display-name=%{GROUP} user=omero python-path=/usr/lib64/python2.6/site-packages/Ice:/home/omero/OMERO.server/lib/python:/home/omero/OMERO.server/lib/fallback:/home/omero/OMERO.server/lib/python/omeroweb
+  WSGIDaemonProcess omeroweb processes=5 threads=1 display-name=%{GROUP} user=omero python-path=/usr/lib64/python2.7/site-packages/Ice:/home/omero/OMERO.server/lib/python:/home/omero/OMERO.server/lib/fallback:/home/omero/OMERO.server/lib/python/omeroweb
```

Comment:
@rleigh-dundee: this is done as described in https://github.com/openmicroscopy/ome-documentation#auto-generated-content. I need to fix this script to account for the FastCGI deprecation anyways. @mtbc: I can look at a way to insert the fact that the file is auto-generated.